common/static/common/js/components/utils/view_utils.js

```diff
@@ -7,8 +7,8 @@
         "common/js/components/views/feedback_prompt"],
     function ($, _, gettext, NotificationView, PromptView) {
         var toggleExpandCollapse, showLoadingIndicator, hideLoadingIndicator, confirmThenRunOperation,
-            runOperationShowingMessage, disableElementWhileRunning, getScrollOffset, setScrollOffset,
-            setScrollTop, redirect, reload, hasChangedAttributes, deleteNotificationHandler,
+            runOperationShowingMessage, withDisabledElement, disableElementWhileRunning,
+            getScrollOffset, setScrollOffset, setScrollTop, redirect, reload, hasChangedAttributes, deleteNotificationHandler,
```

Comment:
If disableElementWhileRunning isn't used anywhere (except its unit test), and it wasn't useful for this case, I would propose that we delete it. But I don't know the issues about why it couldn't be used... so I leave that up to your judgement, @efischer19, as to whether or not the method is potentially useful.